src/generators/python/renderers/ClassRenderer.ts

```diff
@@ -76,13 +76,20 @@ export const PYTHON_DEFAULT_CLASS_PRESET: ClassPresetType<PythonOptions> = {
   },
   ctor({renderer, model}) {
     const properties = model.properties || {};
-    const assigments = Object.values(properties).map((property) => {
-      if (!property.required) {
-        return `if hasattr(input, '${property.propertyName}'):\n\tself._${property.propertyName} = input.${property.propertyName}`;
-      }
-      return `self._${property.propertyName} = input.${property.propertyName}`;
-    });
-    const body = renderer.renderBlock(assigments);
+    let body = '';
+    if (Object.keys(properties).length > 0) {
+      const assigments = Object.values(properties).map((property) => {
+        if (!property.required) {
+          return `if hasattr(input, '${property.propertyName}'):\n\tself._${property.propertyName} = input.${property.propertyName}`;
+        }
+        return `self._${property.propertyName} = input.${property.propertyName}`;
+      });
+      body = renderer.renderBlock(assigments);
+    } else {
+      body = `"""
+No properties
+"""`;
```

Comment:
Do we need that comment?

Reply:
Not necessarily that comment, but yes, otherwise compilation fails 😅

Reply:
Maybe `pass` is better? 🤔 https://www.programiz.com/python-programming/pass-statement

Reply:
No, comment is good :) I don't have a problem with it, I only asked.